### PR TITLE
Update datadog secret in BGS

### PR DESCRIPTION
**Pull request by ponny-m**
Updated BGS helm chart to retrieve secret details for datadog user in all environments.

### CODE REVIEW

**Review by chengjie8 — APPROVED** [review]
LGTM

File: helm/svc-bgs-api/values.yaml [hunk 1]
```diff
@@ -54,3 +54,9 @@ bip:
   # TODO: Consider moving these URL settings to application-*.yml files
   bipClaimUrl:    claims-uat.stage.bip.va.gov/api/v1
   bipEvidenceUrl: vefs-claimevidence-uat.stage.bip.va.gov/api/v1/rest
+
+secrets:
+  rabbitmq:
+    name: vro-dd-rabbitmq-pass
+    key: rabbitmq_pass
```
Comment (lisac) [inline helm/svc-bgs-api/values.yaml:61]: it looks like `secrets.rabbitmq.name` and `secrets.rabbitmq.key` are consistent, regardless of environment, is that correct? if so, then I'd recommend removing the overrides in the env-specific files (eg `values-for-sandbox.yaml`).

File: scripts/image_vars.src [hunk 2]
```diff
@@ -49,7 +49,7 @@ imageTagExists(){
 export postgres_GRADLE_IMG="va/abd_vro-postgres"
 export postgres_IMG="vro-postgres"
 
-export rabbitmq_GRADLE_IMG="va/abd_vro-rabbitmq"
+export rabbitmq_GRADLE_IMG="va/abd_rabbitmq"
```
Comment (lisac) [inline scripts/image_vars.src]: to double-check: is this change in scope of this PR?

Comment (ponny-m) [reply scripts/image_vars.src]: Nope. Took it out

File: helm/svc-bgs-api/values.yaml (hunk 1 above)
Comment (ponny-m) [reply helm/svc-bgs-api/values.yaml:61]: Yes but when we had it in the deployment.yaml it wasn't being pulled/mounted in other environments.
